contracts/script/multisigTransactionProposals/safeSDK/modifyProverModeProposal.ts

```diff
@@ -48,6 +59,21 @@ async function main() {
   }
 }
 
+function processCommandLineArguments(): string {
+  const args = process.argv.slice(2); // Remove the first two args (node command and script name)
```

Comment:
Do we want to possibly consider command line flags for this script?

Will this always be invoked with this form:

```sh
ts-node <script-name> <command>
```

Or might it be invoked in another way, like a node-binary that could be invoked with `npx` or `tsx`?

Reply:
This script is going to be run rarely, but can also be ran with npx. What is your preference?

Reply:
Ah.  I just wanted to make sure that the calculation of the args here would always be correct.  Just to make sure that this is indeed where the args will start.